code/modules/food_and_drinks/food/snacks_vend.dm

```diff
@@ -102,3 +102,13 @@
 	filling_color = "#97ee63"
 	tastes = list("pure electricity" = 3, "fitness" = 2)
 	foodtype = TOXIC
+	
+/obj/item/reagent_containers/food/snacks/Gobstopper
+	name = "Nuclear Gobstopper"
+	icon_state = "gobstopper"
+	desc = "A Domestic Terrorist’s favorite candy! Disk not included."
+	trash = /obj/item/trash/gobstopper
+	list_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/sugar = 3, /datum/reagent/medicine/inacusiate = 1)
```

Comment:
Haha no. Not happening, get your powergame chems out of that

Reply:
The hearing restoration chem, if that wasnt clear

Reply:
i dont care about it either way originally it was suggested to be stimualnts but that is bullshit, also you know alex carrots have this same chemical and are also mass producable

Reply:
what would be a gool alternative?

Reply:
Yeah I would love to know an alternative too

Reply:
we dont have any food that has inacusiate afaik

this is a good change

Reply:
we dont have any food that has inacusiate afaik

this is a good change

Reply:
pOwErGaMe CheMs

Of all the retarded, powergamey shit you've merged without a second thought alexcar, you're throwing a fit over fucking inacusiate? The incredibly easy to mass produce drug that just fixes ear damage? That?

Fuck that, this is fine.

Reply:
Yeah ear damage heals on its own, and if your ears are permanently broken from explosions, you are probably missing a goddamn leg.

Reply:
The thing we need is not even less dependency on medbay, medbay already has a problem of everyone doing their jobs at their place. Having such a specific heal in a commonly available yet obscure item will result in people beelining for vendors when they are deaf instead of you know, going to medbay. Or worse they'll start packing them with them. 

The fact the drug is easy to create does not excuse the drug being common, charcoal is easy to make yet it is quite hard to find outside of medbay and that's intentional.  Same can be said about inacusite, it's a very specific chem and will give an edge on people who know about this obscure item on those who dont. The edge isnt a combat buff but a gameplay one.

Reply:
@alexkar598 what would be a better chem, in your opinion?

Reply:
Nobody cares about being deaf because it wears off. Nobody goes to medbay because they're deaf.

Reply:
> The thing we need is not even less dependency on medbay, medbay already has a problem of everyone doing their jobs at their place. Having such a specific heal in a commonly available yet obscure item will result in people beelining for vendors when they are deaf instead of you know, going to medbay. Or worse they'll start packing them with them.
> 
> The fact the drug is easy to create does not excuse the drug being common, charcoal is easy to make yet it is quite hard to find outside of medbay and that's intentional. Same can be said about inacusite, it's a very specific chem and will give an edge on people who know about this obscure item on those who dont. The edge isnt a combat buff but a gameplay one.

so can we get a alternate chemical please?

Reply:
Dunno why it needs chems inside it but Salbutamol would be okay I guess. Maybe anti rads as they're pretty much useless without having 50 million units of it

Reply:
yeah anti rads sounds good

Reply:
will do